test/provisioner/provision_ibmcloud.go

```diff
@@ -300,7 +332,28 @@ func (p *IBMCloudProvisioner) CreateCluster(ctx context.Context, cfg *envconf.Co
 		return err
 	}
 
-	// TODO, wait cluter ready, it takes long time
+	clusterReady := false
+	waitMinutes := 50
+	ibmcloudTrace(fmt.Sprintf("Waiting cluster %s available.\n", IBMCloudProps.ClusterName))
```

Comment:
nit:
```suggestion
	ibmcloudTrace(fmt.Sprintf("Waiting for cluster %s to be available...\n", IBMCloudProps.ClusterName))
```

Reply:
I think there is an argument that some of these messages e.g. 
`Creating cluster sh-e2e-test.`
`Waiting cluster sh-e2e-test available.`
`Slept x seconds...` or there new variants should be printed even if DEBUG isn't set to true, otherwise it looks like the command has just hung for ~20mins?

Reply:
That's right, thank you @stevenhorsman , I'll just use logger.